src/app.gresource.xml

```diff
@@ -37,5 +37,8 @@
     <file>Library/demos/Custom Widget/main.ui</file>
     <file>Library/demos/Custom Widget/main.js</file>
     <file>Library/demos/Custom Widget/main.json</file>
+    <file>Library/demos/Button/main.blp</file>
+    <file>Library/demos/Button/main.json</file>
+    <file>Library/demos/Button/main.js</file>
```

Comment:
this is not needed anymore (you will have to merge main into your branch)

https://github.com/sonnyp/Workbench/pull/200